src/main/java/org/opensearch/flowframework/rest/RestProvisionWorkflowAction.java

```diff
@@ -84,7 +84,7 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
                 throw new FlowFrameworkException("workflow_id cannot be null", RestStatus.BAD_REQUEST);
             }
             // Create request and provision
-            WorkflowRequest workflowRequest = new WorkflowRequest(workflowId, null);
+            WorkflowRequest workflowRequest = new WorkflowRequest(workflowId, null, null, null);
```

Comment:
Just a nit, can we overload the `WorkflowRequest` constructor so that we dont have to pass nulls here

Reply:
Shouldn't be a nit. Good point.